ui_testing/testcases/basic_tests/test06_orders.py

```diff
@@ -2296,3 +2298,61 @@ def test065_download_suborder_sheet_for_single_order(self):
                                   f"{str(fixed_sheet_row_data)} : {str(formatted_orders[index])}")
             for item in formatted_orders[index]:
                 self.assertIn(item, fixed_sheet_row_data)
+
+    @skip("https://modeso.atlassian.net/browse/LIMSA-236")
```

Comment:
you created new branch from your previous branch.
So, all functions you defined in last PR repeated her.
Now, you can remove this function from this PR. 
And in next task, make sure you are on master branch when you create new branch